src/data/queries/vamcSystemVaPolice.ts

```diff
@@ -96,9 +102,25 @@ export const formatter: QueryFormatter<
       id: entity.field_phone_numbers_paragraph?.[0]?.id || '',
       type: 'paragraph--phone_number',
     },
-    // phoneNumber: queries.formatData(
-    //       'paragraph--phone_number',
-    //       entity.field_phone_numbers_paragraph[0]
-    //     ),
+    policeReport: {
+      id: entity.field_cc_police_report?.[0]?.target_id || '',
```

Comment:
This is always a string, right? Otherwise if the id were `0`, this could end up being `''` because of `||` vs `??`

Reply:
Yep, we really needed that `??` operator in JavaScript because of this! I've had so many frontend bugs in the past because of `|| someDefaultValue`